spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/types/AtlasExternalEntityUtilsSuite.scala

```diff
@@ -124,5 +122,28 @@ class AtlasExternalEntityUtilsSuite extends FunSuite with Matchers with WithHive
     pathEntity.getAttribute(AtlasClient.REFERENCEABLE_ATTRIBUTE_NAME) should be (
       tempFile.toURI.toString)
   }
+
+  test("convert jdbc properties to rdbms entity") {
+    val tableName = "employee"
+    val rdbmsEntity = external.rdbmsTableToEntity("jdbc:mysql://localhost:3306/default", tableName)
```

Comment:
Let's have various JDBC URLs as test cases to make sure what we are supporting. Even better to add some dummy properties (if JDBC URL format of such DBMS supports it) to show we are properly extracting database name.